products/scala.md

```diff
@@ -107,12 +115,19 @@ Scala’s primary platform is the Java Virtual Machine (JVM).
 
 | JDK version | Minimum Scala versions         |
 |-------------|--------------------------------|
-| 21 (ea)     | 3.3.1, 2.13.11, 2.12.18        |
+| 22 (ea)     | 3.3.4*, 2.13.12, 2.12.19       |
+| 21 (LTS)    | 3.3.1, 2.13.11, 2.12.18        |
 | 20          | 3.3.0, 2.13.11, 2.12.18        |
 | 19          | 3.2.0, 2.13.9, 2.12.16         |
 | 18          | 3.1.3, 2.13.7, 2.12.15         |
 | 17 (LTS)    | 3.0.0, 2.13.6, 2.12.15         |
 | 11 (LTS)    | 3.0.0, 2.13.0, 2.12.4, 2.11.12 |
 | 8 (LTS)     | 3.0.0, 2.13.0, 2.12.0, 2.11.0  |
 
-Using the latest patch version is always recommended.
+* = forthcoming; support available in nightly builds
+
+Even when a version combination isn’t listed as supported, most features might still work.
+
+Using latest patch version of Scala is always recommended.
+
+Lightbend offers commercial support for Scala 2. The linked page includes contact information for inquiring about supported and recommended versions.
```

Comment:
The linked page ?

Reply:
The [scala-website](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html#scala-compatibility-table) links to https://www.lightbend.com/lightbend-platform-subscription, which now redirects (301) to https://www.lightbend.com/akka/pricing and doesn't mention Scala any more. I could not find a cached copy either.

https://www.lightbend.com/legal/support doesn't mention Scala either, but https://developer.lightbend.com/docs/introduction/getting-help/support-terminology.html lists Scala as a supported product (but links to the unavailable platform subscription link).

Since there is no clear lightbend doc that confirms support, let's drop this?
```suggestion
Lightbend offers commercial support for Scala 2. The linked page includes contact information for inquiring about supported and recommended versions.
```

Reply:
I just went ahead dropping this note entirely.